### DISCUSSION TITLE
Is there a way to output table cells in a for loop?

### DISCUSSION THREAD

**Discussion by cds-internationalist** [typst/typst#830] [Q&A] [answered]
Maybe this is trivial, but having a hard time finding details in the documentation. Table cells (among other structures) are separated by commas in code - is there a way to output table cells in a for loop? My feeble attempts have so far just resulted in a error or outputting commas to the text rather than code.

**Comment by johannes-wolf** [comment, marked as answer]
You can use the spread operator `..` like this:
```typst
#table(columns: 2,
..(for a in range(1, 11) { ([#a], ) }))
                           /*   ^-- Note the trailing comma to create an array with one item */
```
All arrays returned by `for` are joined into one single array, which then is spread using `..`.
